c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -643,7 +665,6 @@ node_modules/ngeo/buildtools/mako_build.json: .build/node_modules.timestamp
 		--var 'src=$(PACKAGE)/static-ngeo/js/**/*.js' \
 		--var src_set=contribs_gmf \
 		--var entry_point=$(PACKAGE)_$* \
-		--var js=$(PACKAGE)/static-ngeo/js/$*.js \
```

Comment:
I cannot really remember if we did that on Wednesday or if this is only a nasty copy-paste removal....